spec/unit/provider/package/pacman_spec.rb

```diff
@@ -10,6 +10,12 @@
   let(:resource) { Puppet::Type.type(:package).new(:name => 'package', :provider => 'pacman') }
   let(:provider) { described_class.new(resource) }
 
+  if Puppet.features.microsoft_windows?
```

Comment:
Is this the best thing for us to do on Windows?  What about running these tests in parallel makes this necessary?

Probably the fastest way to create / exit a process on Windows is `cmd.exe /c "exit 0"` - but I'm not quite understanding why this is necessary for `$CHILD_STATUS` ... is there no other way to stub that constant?